jvm-packages/xgboost4j/src/native/xgboost4j.cpp

```diff
@@ -22,6 +22,11 @@
 #include <vector>
 #include <string>
 
+#define JVM_CHECK_CALL(__errcode)                   \
```

Comment:
This JVM stuff needs to be separated and reviewed by JVM contributers.

Reply:
It comes up when enforcing the number of columns.  Will split it up.